util/chef-install.sh

```diff
@@ -275,12 +275,8 @@ install_file() {
       ;;
     "deb")
       echo "installing with dpkg..."
-      next_wait=0
-      until dpkg -i "$2" || [ ${next_wait} -ge 10 ] ; do
-        echo "Retrying dpkg -i $2 ..."
-        sleep $((next_wait=next_wait+1))
-      done
-      [ ${next_wait} -lt 10 ]
+      flock $(apt-config shell StateDir Dir::State/d | sed -r "s/.*'(.*\/?)'$/\1/")daily_lock systemctl stop apt-daily.timer apt-daily.service
```

Comment:
Same comment as [https://github.com/aws/aws-parallelcluster-cookbook/pull/616#discussion_r425981332](here): should there be a  `/` before `daily_lock`?

Reply:
Ah nevermind, from the apt-config man page I saw that the trailing `/d` ensures the trailing`/` is there.